openfl/component/director/director.py

```diff
@@ -230,6 +230,7 @@ async def _run_aggregator(
         finally:
             os.chdir(cwd)
             shutil.rmtree(experiment_name)
+            os.remove(data_file_name)
```

Comment:
Do we need `data_file_name` here just to remove the archive? We could do it after line 69 `create_workspace`

Reply:
We use it to send to envoys. Inside `create_workspace` we create experiment workspace for aggregator, but the director has to send experiment archive to envoy and it is hardly a good idea get it form aggregator workspace. We can remove it earlier, after sending to all envoys, but it makes sense to keep it until the end of the experiment like director experiment context.